engine/execution/state/bootstrap/bootstrap.go

```diff
@@ -51,6 +51,8 @@ func (b *Bootstrapper) BootstrapLedger(
 		fvm.WithLogger(b.logger),
 		fvm.WithMaxStateInteractionSize(ledgerIntractionLimitNeededForBootstrapping),
 		fvm.WithChain(chain),
+		// TODO (JanezP): move this deeper
+		fvm.WithEVMEnabled(true),
```

Comment:
is this still a todo for this PR or for a later time?

Reply:
Thats for a different PR, because a lot of test code needs to be touched, and this PR is large already. I'll open an issue.